tests/Feature/SearchTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * We don't care for unhandled exceptions in tests.
+ * It is the nature of a test to throw an exception.
+ * Without this suppression we had 100+ Linter warning in this file which
+ * don't help anything.
+ *
+ * @noinspection PhpDocMissingThrowsInspection
+ * @noinspection PhpUnhandledExceptionInspection
+ */
+
+namespace Tests\Feature;
+
+use App\Facades\AccessControl;
+use Tests\Feature\Lib\AlbumsUnitTest;
+use Tests\Feature\Lib\PhotosUnitTest;
+use Tests\TestCase;
+
+class SearchTest extends TestCase
+{
+	/**
+	 * Test searching is working.
+	 *
+	 * @return void
+	 */
+	public function testSearch(): void
```

Comment:
Again, please split this test methods into multiple test methods, see previous comment. If I am not mistaken, I am able to identify at least three test cases

- `testSearchPhotoByTitle`
- `testSearchPhotoByTag`
- `testSearchAlbumByTag`

This should at least be three different test methods with proper names.